src/utils/ResultListUtils.ts

```diff
@@ -40,7 +40,10 @@ export class ResultListUtils {
 
   private static getActiveResultList(root: HTMLElement) {
```

Comment:
`getActiveResultList` makes me expect it to return *any* kind of result list. I think we should rename it to something like `getMainResultList`, `getPageControllingResultList` or something better.